pynamodb/models.py

```diff
@@ -357,6 +361,52 @@ def update_item(self, attribute, value=None, action=None, conditional_operator=N
                 setattr(self, name, attr.deserialize(value.get(ATTR_TYPE_MAP[attr.attr_type])))
         return data
 
+
+    def update(self, attributes,  conditional_operator=None, **expected_values):
```

Comment:
@jmphilli you might need to pass a value for `action` beside the `value`. That would mean a bunch of small dictionaries like `attribute_name={'value': 1, 'action': 'add'}`.